### PR TITLE
Add support for Microsoft Edge in Mac OS.

### PR DESCRIPTION
**Pull request by elsazac**
In Eclipse->Settings->General->Web Browser

1.As cleanup removing browsers Camino and Internet Explorer, as it is no longer supported in Mac OS. 
2.Adding support for Microsoft Edge in Mac OS.

/cc @lshanmug

### CODE REVIEW

**Comment by vogella** [comment]
@BeckerWdf can you test?

**Comment by BeckerWdf** [comment]
> @BeckerWdf can you test?

Sure.
I just installed Edge on my Mac. :see_no_evil:

I created a project with a simple html file. Choosing "open > web browser" from the context menu opens edge when "edge" is selected in the prefereces. It openes "chrome" when "chrom" is selected in the preferences.
The "Browser" view in contrast does not change behaviour. It always uses "safari" on macos. But I think this is how this always worked.

So function wise this looks good to me.

File: bundles/org.eclipse.ui.browser/plugin.xml [hunk 1]
```diff
@@ -201,13 +201,13 @@
          executable="Google Chrome">
          <location>Applications/Google Chrome.app</location>
       </browser>
-      <browser
+<!--  <browser
```
Comment (BeckerWdf) [inline bundles/org.eclipse.ui.browser/plugin.xml]: I would prefer that you delete this instead of commenting it out.

**Review by BeckerWdf — CHANGES_REQUESTED** [review]

File: bundles/org.eclipse.ui.browser/plugin.xml [hunk 2]
```diff
@@ -216,12 +216,19 @@
          factoryclass="org.eclipse.ui.internal.browser.macosx.SafariBrowserFactory">
          <location>Applications/Safari.app</location>
       </browser>
-      <browser
+<!--  <browser
```
Comment (BeckerWdf) [inline bundles/org.eclipse.ui.browser/plugin.xml]: same here: pls. delete.
In addition. Also delete browserInternetExplorer and browserCamino in plugin.properties

Comment (elsazac) [reply bundles/org.eclipse.ui.browser/plugin.xml]: @BeckerWdf Deleted both camino and Internet Explorer.

Comment (elsazac) [reply bundles/org.eclipse.ui.browser/plugin.xml]: @BeckerWdf Deleted both from Plugin.properties and Plugin.xml.Can you have a look at it?Let me know if anything else is required.

**Review by BeckerWdf — APPROVED** [review]

**Comment by vogella** [comment]
AFAICS the tests errors are unrelated and covered by https://github.com/eclipse-platform/eclipse.platform.ui/issues/665. Merging based on @BeckerWdf review.

Thanks [elsazac](https://github.com/elsazac)

**Pull request merged**